PlannTer/UI/Inputs/DropdownPicker.swift

```diff
@@ -0,0 +1,88 @@
+import SwiftUI
+
+struct DropdownPicker: View {
+    let title: String
+    let options: [String]
+    
+    @State private var isExpanded = false
+    @State private var selectedOption: String
+    
+    init(title: String, options: [String]) {
+        self.title = title
+        self.options = options
+        self._selectedOption = State(initialValue: options.first ?? "")
+    }
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(title)
+                .font(.caption)
```

Comment:
font probably should stay as note - the little size 16 we defined. At least it's this way in figma